common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -66,12 +78,14 @@
             ].join(''),
 
             template: [
-                '<ol id="transcript-captions" class="subtitles" aria-label="',
+                '<div class="subtitles" role="region" aria-label="',
                     /* jshint maxlen:200 */
                     gettext('Activating an item in this group will spool the video to the corresponding time point. To skip transcript, go to previous item.'),
                     '">',
-                    '<li></li>',
-                '</ol>'
+                    '<ol id="transcript-captions" class="subtitles-menu">',
+                        '<li></li>',
```

Comment:
It was there previously so I kept it. However I just tried removing it and things seem to work okay. Removing!